pyuvdata/uvdata/mir_parser.py

```diff
@@ -1642,6 +1990,73 @@ def reset(self):
 
         self.unload_data()
 
+    def save_mask(self, mask_name: str = None, overwrite=False):
+        """
+        Save masks for later use.
+
+        This method will store selection masks, that can later be reloaded. This can
+        be useful for temporarily down-selecting data for various operations (allowing
+        one to restore the old selection mask after that is complete).
+
+        Parameters
+        ----------
+        mask_name : str
+            Name under which to store the mask. No default.
+        overwrite : bool
+            If set to False and attempting to save a mask whose name is already stored,
+            an error is thrown. If set to True, and existing masks stored under the name
+            given will be overwritten. Default is False.
+        """
+        if (mask_name in self._stored_masks) and not overwrite:
+            raise ValueError(
+                "There already exists a stored set of masks with the name %s, "
+                "either change the name or set overwrite=True" % mask_name
+            )
+
+        if not isinstance(mask_name, str):
+            raise ValueError("mask_name must be a string.")
+
+        for name in self._metadata_attrs:
+            self._metadata_attrs[name]
```

Comment:
What is this line/loop doing? Just returning the value from that key in the dict?

Reply:
I... don't know. It's clearly not doing anything useful, I think I may have started writing what became the loop down below and somehow forgot that I had this started above. Opps.

Reply:
Removed in b36ea1b702e7a8f6a5a2c20b4fd257bf24c94afd